filebeat/module/nats/module.yml

```diff
@@ -0,0 +1,3 @@
+dashboards:
```

Comment:
Can you remove this and add it in the PR with the dashboard?

Reply:
pretty sure